flanker/addresslib/address.py

```diff
@@ -284,23 +286,30 @@ def validate_address(addr_spec, metrics=False):
     # sanity check
     if addr_spec is None:
         return None, mtimes
-
-    # preparse address into its parts and perform any ESP specific pre-parsing
-    addr_parts = preparse_address(addr_spec)
-    if addr_parts is None:
-        _log.warning('failed preparse check for %s', addr_spec)
+    if '@' not in addr_spec:
         return None, mtimes
 
     # run parser against address
     bstart = time()
-    paddr = parse('@'.join(addr_parts), addr_spec_only=True, strict=True)
+    paddr = parse(addr_spec, addr_spec_only=True, strict=True)
     mtimes['parsing'] = time() - bstart
     if paddr is None:
         _log.warning('failed parse check for %s', addr_spec)
         return None, mtimes
 
+    # lookup the TLD
+    bstart = time()
+    tld = get_tld(paddr.hostname, fail_silently=True, fix_protocol=True)
+    mtimes['tld_lookup'] = time() - bstart
+    if tld is None:
+        _log.warning('failed tld check for %s', addr_spec)
```

Comment:
do we really need the warning here? It just floods logs

Reply:
I added that to be consistent with the other checks. I'll move it to info.

Reply:
I would move it to debug. Our logs are overloaded  with `[flanker.addresslib.address] relaxed parsing is not available for discrete lists, ignoring` already

Reply:
Done.